tools/clang/test/DXC/strip_dbg.test

```diff
@@ -1,7 +1,11 @@
 // /Fd implies /Qstrip_debug ; path with / produces auto hash-named .pdb file
-// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Fd ./ /Fo %t.strip
+
+// RUN: mkdir -p %T/pdbdir
```

Comment:
The issue with using `%T` instead of basing it off of `%t` is that it's not unique for this test.

Reply:
Fixed.